### PR TITLE
MS. Selden Supra 53: new description

### PR DESCRIPTION
**Pull request by adunning**

### CODE REVIEW

File: collections/Selden_supra/MS_Selden_Supra_53.xml [hunk 1]
```diff
@@ -38,43 +37,177 @@
...
-                     <title key="work_2720">The Dance of Death</title>
-                     <textLang mainLang="en">English</textLang>
+                     <title key="work_2720">Dance of Macabre</title>
+                     <incipit>O ȝee folkes harde hertid as a stone <lb/>Wich to þe worlde haue al ȝour aduertence</incipit>
+                     <explicit>Of her tunge I haue no suffisaunce <lb/>Her corious metris in englisshe to translate</explicit>
+                     <finalRubric>Here endith the daunce of deeth</finalRubric>
+                     <bibl type="repertory"><ref target="https://www.dimev.net/record.php?recID=4107"><title>DIMEV</title> 4107</ref></bibl>
+                  </msItem>
+                  <msItem>
+                     <note>Items added in a later hand, third quarter of the 15 century.</note>
```
Comment (holfordm) [inline collections/Selden_supra/MS_Selden_Supra_53.xml:141]: typo, 15 for 15th

File: collections/Selden_supra/MS_Selden_Supra_53.xml [hunk 2]
```diff
@@ -38,43 +37,177 @@
...
+                        <incipit>This Emperour above expressid. is oure lorde god þat hath .iii. sones</incipit>
+                        <explicit>þe kyngdom of hevene To wiche god bringe vs alle. Amen. Amen. Amen.</explicit>
+                     </msItem>
+                  </msItem>
+                  <msItem>
+                     <locus>(fols. 148r–158v)</locus>
                      <author key="person_22158282">John Lydgate</author>
-                     <title key="work_2720">The Dance of Death</title>
-                     <textLang mainLang="en">English</textLang>
+                     <title key="work_2720">Dance of Macabre</title>
```
Comment (holfordm) [inline collections/Selden_supra/MS_Selden_Supra_53.xml:134]: should this title be "Dance Macabre"?

File: collections/Selden_supra/MS_Selden_Supra_53.xml [hunk 3]
```diff
@@ -38,43 +37,177 @@
...
+                     <incipit>Nowe herkene a doctrine substancial <lb/>ffirste howe lerne to die telle wole I</incipit>
+                     <explicit>That it may haue þe lyf þat hath noon ende</explicit>
+                     <finalRubric>Here endith to lerne to die⸝ And bigynneth a prolog on þe ixᵉ leson þat is redde on alle halwen day</finalRubric>
+                     <note>Missing stanzas 1–3.</note>
+                     <bibl type="repertory"><ref target="https://www.dimev.net/record.php?recID=4879"><title>DIMEV</title> 4879</ref></bibl>
+                     <msItem>
+                        <locus>(fol. 133r)</locus>
+                        <incipit>To oþer þre partes⸝ edich in þe book</incipit>
+                        <explicit>That for to comprehende þat gladnesse <lb/>Verrely no wit may ne tounge expresse</explicit>
+                     </msItem>
```
Comment (holfordm) [inline collections/Selden_supra/MS_Selden_Supra_53.xml:98]: query what this is and whether the incipit is correct. DIMEV lists the incipit (https://www.dimev.net/Results.php?startsWith=T-t no. 425) but with a crossreference I can't make sense of. Does the print NIMEV help?

File: collections/Selden_supra/MS_Selden_Supra_53.xml [hunk 4]
```diff
@@ -38,43 +37,177 @@
...
+                     <bibl type="repertory"><ref target="https://www.dimev.net/record.php?recID=3581"><title>DIMEV</title> 3581</ref></bibl>
                   </msItem>
-                  <msItem n="2" xml:id="MS_Selden_Supra_53-item2">
+                  <msItem>
+                     <locus>(fols. 76r–83v)</locus>
+                     <author key="person_9902409">Thomas Hoccleve</author>
+                     <title key="work_4805">Complaint</title>
+                     <msItem>
+                        <incipit>Aftir tat heruest Inned had hise sheues <lb/>And that the groun sesoun of Mihelmesse</incipit>
+                        <explicit>I braste once⸝ on þe morede⸝ and þus began.</explicit>
```
Comment (holfordm) [inline collections/Selden_supra/MS_Selden_Supra_53.xml:57]: "tat" ok?

File: collections/Selden_supra/MS_Selden_Supra_53.xml [hunk 5]
```diff
@@ -38,43 +37,177 @@
...
+                  <textLang mainLang="en" otherLangs="la">Middle English with Latin</textLang>
+                  <msItem>
+                     <locus>(fols. 1r–76r)</locus>
                      <author key="person_9902409">Thomas Hoccleve</author>
-                     <title key="work_4809">poems</title>
-                     <textLang mainLang="en">English</textLang>
+                     <title key="work_4806">De regimine principum</title>
+                     <incipit>Knyȝthood a wake þou slepist to longe <lb/>Thi brother see nowe ny deed for mischeef</incipit>
+                     <explicit>To þee þat al saist of loues feruence <lb/>That knowith he whom no þing is hid fro</explicit>
+                     <note>Lines 1-896 wanting.</note>
```
Comment (holfordm) [inline collections/Selden_supra/MS_Selden_Supra_53.xml:47]: (due to loss of quires at the beginning?)

File: collections/Selden_supra/MS_Selden_Supra_53.xml [hunk 6]
```diff
@@ -38,43 +37,177 @@
                   </altIdentifier>
                </msIdentifier>
                <msContents>
```
Comment (holfordm) [inline collections/Selden_supra/MS_Selden_Supra_53.xml:39]: add heading?

File: collections/Selden_supra/MS_Selden_Supra_53.xml [hunk 7]
```diff
@@ -38,43 +37,177 @@
...
                <history>
                   <origin>
-                     <origDate calendar="Gregorian" when="1430">c. 1430</origDate>
+                     <origDate calendar="Gregorian" notAfter="1450" notBefore="1425">15th century, second quarter</origDate>
                      <origPlace>
                         <country key="place_7002445">English</country>
                      </origPlace>
                   </origin>
-                  <provenance notAfter="1654" notBefore="1584" resp="#MMM"><persName key="person_51728429" role="fmo">John Selden, 1584–1654</persName></provenance>
+                  <provenance>Richard Burnett (16th century): evidently responsible for marginal annotion after fol. 83r.</provenance>
```
Comment (holfordm) [inline collections/Selden_supra/MS_Selden_Supra_53.xml:203]: authority record?

File: collections/Selden_supra/MS_Selden_Supra_53.xml [hunk 8]
```diff
@@ -38,43 +37,177 @@
...
                   <objectDesc form="codex">
                      <supportDesc material="perg">
                         <support>parchment</support>
+                        <extent>ii (modern paper flyleaves) + 158 + 1 (medieval flyleaf) + ii (paper flyleaves, foliated 160 and 161) leaves <dimensions type="leaf" unit="mm">
+                              <height>253</height>
+                              <width>188</width>
+                           </dimensions>
+                        </extent>
+                        <foliation>Modern foliation.</foliation>
+                        <collation>Bound mainly gatherings of 8. <signatures>Signatures in the lower margin on line for catchwords, upper case letters plus Roman numerals, in red ink: first half of quire, beginning C.i. on fol. 1 and continuing through 'Z' for last quire.</signatures>
```
Comment (holfordm) [inline collections/Selden_supra/MS_Selden_Supra_53.xml:168]: missing "in" (Bound in ...)

File: collections/Selden_supra/MS_Selden_Supra_53.xml (hunk 3 above)
Comment (adunning) [reply collections/Selden_supra/MS_Selden_Supra_53.xml:98]: DIMEV treat this prologue as part of no. 4879 – though I am not sure that is really correct.

**Pull request merged**